### PR TITLE
Small fixes and test

### PR DESCRIPTION
**Pull request by jafu888**
Checks for a few things:
performance on nested MotionLayout
Fast Transition with NestedScrollView
ConstraintSet Gon

### CODE REVIEW

File: constraintlayout/.idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (camaelon) [inline constraintlayout/.idea/codeStyles/Project.xml:1]: do you want to add this file??

File: projects/MotionLayoutVerification/app/src/main/java/android/support/constraint/app/VerificationActivity.java [hunk 2]
```diff
@@ -76,7 +76,7 @@
 public class VerificationActivity extends AppCompatActivity implements View.OnClickListener {
     private static final String TAG = "Verification00";
     private String KEY = "layout";
-    private static final boolean DEBUG = false;
+    private static final boolean DEBUG = true;
```
Comment (camaelon) [inline projects/MotionLayoutVerification/app/src/main/java/android/support/constraint/app/VerificationActivity.java:79]: is that intended?

**Review by camaelon — APPROVED** [review]

File: constraintlayout/.idea/codeStyles/Project.xml (hunk 1 above)
Comment (jafu888) [reply constraintlayout/.idea/codeStyles/Project.xml:1]: I will remove them in next PL

File: projects/MotionLayoutVerification/app/src/main/java/android/support/constraint/app/VerificationActivity.java (hunk 2 above)
Comment (jafu888) [reply projects/MotionLayoutVerification/app/src/main/java/android/support/constraint/app/VerificationActivity.java:79]: I will remove them in next PL

**Pull request merged**